scalajvm/app/controllers/Approving.scala

```diff
@@ -38,13 +41,13 @@ object Approving extends Controller {
                 val approvedQuote = QuoteQueries().getQuoteById(QuoteQueries().insertQuote(form.content, suggestedQuote.source)).get
                 val approvers = ApproverQueries().getAllApprovers
                 SuggestedQuoteQueries().deleteSuggestedQuoteByToken(form.token)
-                Notifications.notifyApproversAboutApprovedQuote(approvers, suggestedQuote, approvedQuote)
+                notifications.notifyApproversAboutApprovedQuote(approvers, suggestedQuote, approvedQuote)
```

Comment:
`Notifications` used to be an `object`, but it's a class because of DI. In reality, it's used more like a namespace than a class; is there a better way to restructure `Notifications` so we don't have to instantiate it?

Reply:
Notifications is just plain object with bunch of functions. Supply configuration options like `approvalSmtpHost` directly whether possible and use `import Notifications._` for convenience here.

Reply:
Thanks! Applied in https://github.com/codingteam/loglist/pull/225/commits/131d0454e255d6994fa73bb0365788717c185f7e

I decided against `import Notifications._` because the explicit mention of an object doesn't bother me that much (and apparently didn't bother anyone else, since it survived until today).